Samples/MediaPlayerCS/MainPage.xaml.cs

```diff
@@ -91,43 +91,43 @@ private void OnUriBoxKeyUp(object sender, KeyRoutedEventArgs args)
 			}
 		}
 
-		private void OpenUri(String uri)
-		{
+		private void OpenUri(String uri)
+		{
 			OpenMedia((mss, config) =>
 			{
-				FFmpegInteropMSS.CreateFromUri(uri, mss, config);
-			});
+				FFmpegInteropMSS.InitializeFromUri(uri, mss, config);
+			});
 		}
 
-		private void OpenMedia(Action<MediaStreamSource, FFmpegInteropMSSConfig> createFunc)
-		{
-			// Stop any media currently playing
-			mediaElement.Stop();
-
-			try
-			{
-				// Create the media source
-				IActivationFactory mssFactory = WindowsRuntimeMarshal.GetActivationFactory(typeof(MediaStreamSource));
-				MediaStreamSource mss = mssFactory.ActivateInstance() as MediaStreamSource;
-
-				var config = new FFmpegInteropMSSConfig
-				{
-					ForceAudioDecode = toggleSwitchAudioDecode.IsOn,
-					ForceVideoDecode = toggleSwitchVideoDecode.IsOn
-				};
-
-				createFunc(mss, config);
-
-				// Set the MSS as the media element's source
-				mediaElement.SetMediaStreamSource(mss);
-
-				// Close the control panel
-				splitter.IsPaneOpen = false;
-			}
-			catch (Exception)
-			{
-				OnError("Failed to open media");
-			}
+		private void OpenMedia(Action<MediaStreamSource, FFmpegInteropMSSConfig> createFunc)
```

Comment:
weird that this is showing up as changed. was it just tabbing that was updated?

Reply:
It looks like the file had mixed line endings (CRLF vs LF) and VS Code auto-formatted it for consistency.